samples/non-python-preprocess/devops_pipelines/dependency-update-template.yml

```diff
@@ -0,0 +1,10 @@
+# Pipeline template to update the custom docker image with latest dependencies
+steps:
+- script: |   
+   conda env create -f /home/vsts/ci_dependencies.yml -n ci-latest
```

Comment:
Instead of creating a new environment how about updating the environment called 'ci'? See [here](https://github.com/liupeirong/MLOpsManufacturing/blob/ade1f55e538e33e61bb5c2d49afdf802e94e94d3/samples/non-python-preprocess/devops_pipelines/build_agent/Dockerfile#L39)

Creating a new environment is also valid option, however we need to activate it as well. 

In addition if we update the dependency file copied when the docker image was created, it will be outdated. We have to reference the new dependency yaml from the repository.

Reply:
Thank you for the great feedback! Do you mean to use 'conda env update -f /home/vsts/ci_dependencies.yml' like this way?

Reply:
you should reference [this yml file](https://github.com/liupeirong/MLOpsManufacturing/blob/eunk/40_amlsdk_mocking/samples/non-python-preprocess/local_development/dev_dependencies.yml)